src/theme.js

```diff
@@ -3,6 +3,16 @@ import icons from '../icons.json'
 
 const createMediaQuery = n => `@media screen and (min-width:${n}em)`
 
+const addAliases = (arr, aliases) =>
+  aliases.forEach((key, i) =>
+    Object.defineProperty(arr, key, {
+      enumerable: false,
```

Comment:
Is there a specific reason the to-be defined `key` object property not enumerable? Perhaps `for...in loop` or `Object.keys()` type of code can be benefited with the `key` property, if it's enumerable?

Reply:
Basically yeah I didn't want the aliases to show up in the array of breakpoints by accident, in case someone is iterating over it with something like `for...in`. It could be a breaking change if they did that, and I think it's just more intuitive not to change the enumerable behavior... they are aliases after all, not separate breakpoints.

Reply:
This is handy – I wouldn't have thought to define properties on the array